editors/vscode/client/Config.ts

```diff
@@ -141,9 +141,9 @@ interface ConfigInterface {
    *
    * `oxc.configPath`
    *
-   * @default ".oxlintrc.json"
+   * @default undefined
    */
-  configPath: string;
+  configPath: string | undefined;
```

Comment:
Would it make sense to type this as `string | null` here as well, so we are consistent everywhere?